webapp/src/pages/Map.tsx

```diff
@@ -923,177 +898,122 @@ function MapPage() {
-                        ? theme.palette.custom.mapMenuItemBackgroundSelected
-                        : theme.palette.custom.mapMenuBackground,
-                      borderBottom: menuSelection.includes('Configure RSUs')
-                        ? theme.palette.custom.mapMenuItemBorderSelected
-                        : 'none',
-                      ':hover': {
-                        backgroundColor: menuSelection.includes('Configure RSUs')
-                          ? theme.palette.custom.mapMenuItemHoverSelected
-                          : theme.palette.custom.mapMenuItemHoverUnselected,
-                      },
-                    }}
-                  >
-                    <ListItemText primary="Configure RSUs" />
-                  </ListItemButton>
-                </ListItem>
-              )}
-            </List>
-          </AccordionDetails>
-        </Accordion>
-        <Accordion
-          style={{ backgroundColor: alpha(theme.palette.custom.mapMenuBackground, 0.8) }}
-          disableGutters={true}
-          className="menuAccordion"
           sx={{ '&.accordion': { marginBottom: 0 } }}
+          defaultExpanded
+          elevation={0}
         >
           <AccordionSummary
             expandIcon={<ExpandMoreIcon style={{ color: theme.palette.text.primary }} />}
             aria-controls="panel3-content"
             id="panel3-header"
           >
-            <Typography fontSize="medium" color={theme.palette.text.primary}>
+            <Typography className="accordion-header museo-slab" color={theme.palette.text.primary}>
               Filter RSUs
             </Typography>
           </AccordionSummary>
           <AccordionDetails>
-            <ListItem>
-              <DropdownList
-                dataKey="id"
-                textField="name"
-                data={vendorArray}
+            <FormControl fullWidth>
+              <InputLabel htmlFor="vendor">Vendor</InputLabel>
+              <Select
+                id="vendor"
+                label="Vendor"
                 value={selectedVendor}
-                onChange={(value) => {
-                  setVendor(value)
+                defaultValue={selectedVendor}
+                onChange={(event) => {
+                  const vendor = event.target.value as string
+                  console.log(vendor)
+                  setVendor(vendor)
                 }}
-                style={{ width: '100%' }}
-              />
-            </ListItem>
+              >
+                {vendorArray.map((vendor) => (
+                  <MenuItem key={vendor} value={vendor}>
+                    <Typography>{vendor}</Typography>
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+            <Typography sx={{ marginTop: '12px' }}>RSU Status</Typography>
+            <FormControl sx={{ ml: 2, mt: 1 }}>
+              <RadioGroup value={displayType} onChange={handleRsuDisplayTypeChange}>
```

Comment:
question: we start with neither online nor scms status selected, but this is a radiogroup. Should we allow a "no-select" option here as well?

Reply:
This was intended to have 'online' selected by default and is now updated